test/release/3d-playground.html

```diff
@@ -0,0 +1,267 @@
+<div id='map'></div>
```

Comment:
Do you know if this would work with a symbolic link? That would prevent maintaining both when we make changes to it.

Reply:
That seems to work fine: e8639177adc7298b2909a6cfaebbebed97d14b01 and https://77483-8629417-gh.circle-artifacts.com/0/test/release/index.html#page=3d-playground.